### PR TITLE
bpfman: cli: Refactor command handling

### PR DESCRIPTION
**Pull request by dave-tucker**
This moves all clap code into args.rs to allow it to be used via the include! macro in an xtask to generate manpages.

Updates: #703

### CODE REVIEW

File: bpfman/src/cli/args.rs [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+    pub(crate) csi_support: bool,
+}
+
+#[derive(Subcommand, Debug)]
+pub(crate) enum SystemSubcommand {
+    /// Load an eBPF program from a local .o file.
+    Service(ServiceArgs),
+}
+
+/// Parse a single key-value pair
```
Comment (astoycos) [inline bpfman/src/cli/args.rs:323]: nit these should probably live in a `util.rs`

**Review by astoycos — APPROVED** [review]
LGTM

One small nit which I'll make a good-first-issue for I like this organization much better.

File: bpfman/src/cli/args.rs (hunk 1 above)
Comment (anfredette) [reply bpfman/src/cli/args.rs:323]: They could, but they are only useed by the functions in this file, so I think it's okay.

**Review by anfredette — COMMENTED** [review]
LGTM too.

**Pull request merged**